resources/benchmark-runner.mjs

```diff
@@ -346,7 +346,7 @@ export class BenchmarkRunner {
         const iterationEndLabel = "iteration-end";
         for (let i = 0; i < iterationCount; i++) {
             performance.mark(iterationStartLabel);
-            await this._runAllSuites();
+            await this._runAllSuites(!i && params.cacheResources);
```

Comment:
How about putting this as explicit separate `"cacheResources"` step with explicit labels?